src/table/cells/selections.jsx

```diff
@@ -17,22 +17,24 @@ const useStyles = makeStyles({
 
 export function withSelections(CellComponent) {
   const HOC = (props) => {
-    const { cell, selState, selDispatch, className } = props;
+    const { cell, selState, selDispatch, classCellName } = props;
+    window.console.log(classCellName, 'classCellName');
+
     const classes = useStyles();
     const handleMouseUp = (evt) => cell.isDim && selectCell(cell, selState, selDispatch, evt);
     const selectionClass = getSelectionClass(cell, selState);
 
     return (
       <CellComponent
         {...props}
-        className={`${className} ${selectionClass} ${classes[selectionClass]}`}
+        className={`${classCellName} ${selectionClass} ${classes[selectionClass]}`}
```

Comment:
what happens in the case of hovering when making selections? Does it look the same as in the client?

Reply:
it looks the same as the old table and the hover effect is for each row.

Reply:
no, I am wrong. it does not look correct in sense-client. Look into it.